Resources/Views/Home/_previous-conferences.leaf

```diff
@@ -5,38 +5,38 @@
       Conference
     </h2>
     <br>
-    <p class="bold">October 7th, 2021</p>
-    <br>
+    <p class="bold">October 20th, 2022</p>
+    <p><a href="https://www.flickr.com/photos/196979204@N02/albums/72177720303878744" target="_blank">View all</a></p>
     <img src="/img/previous.svg" class="previous-conference-icon">
   </div>
 
   <div class="col-lg-2 col-sm-12 p-0">
-  <a href="/img/jordi.jpg" data-toggle="lightbox">
-    <img src="/img/jordi.jpg" class="previous-conference-image pc-image-md shadow-1-strong conference-image-top-offset bordered" alt="People networking">
+  <a href="/img/0007.swift-leeds-22-jma-photography-min.jpg" data-toggle="lightbox">
```

Comment:
Is there a reason to have these images hosted within the Docker container/repo? Would likely be more sensible to throw them onto AWS, especially as we can apply better compression and caching rules there.